sdk/tables/azure-tables/src/models.ts

```diff
@@ -4,12 +4,34 @@
 import {
   TableQueryOptionalParams,
   TableQueryEntitiesOptionalParams,
+  TableQueryEntitiesWithPartitionAndRowKeyResponse,
+  TableQueryEntitiesResponse,
   TableInsertEntityOptionalParams,
   TableUpdateEntityOptionalParams,
   TableMergeEntityOptionalParams,
   TableSetAccessPolicyOptionalParams
 } from "./generated/models";
 
+/**
+ * Contains response data for the getEntity operation.
+ */
+export type ListEntitiesResponse<T> = Omit<TableQueryEntitiesResponse, "value"> & {
```

Comment:
`Omit` always looks weird when you hover over a response type as the TS language service doesn't do the type math for you. 

I wonder if it would be simpler to just redefine the expected shape here, since the only other property on the response is ` odataMetadata?: string;`

Reply:
That makes sense, I have filed #10534 to replace all our Omits with redefined shapes. Once this PR is merged I'll send one to address this